devon4j-migrate/index.asciidoc

```diff
@@ -0,0 +1,38 @@
+= Migrate a devon4j project to the latest version
+====
+The java commandlet allows to setup Java. Also it supports devon4j. The arguments (devon java «args») are explained by the below details: 
+https://devonfw.com/website/pages/docs/devonfw-ide-usage.asciidoc.html#java.asciidoc
+
+This tutorial mainly focuses on to migrate an old version of devon4j project to the latest version.
+## Prerequisites
+* devonfw IDE
+
+## Learning goals.
+Here in this tutorial you will learn 
+* How to migrate an older version of devon4j project to the latest version.
+
+====
+Install older version of devonfw ( "2020.08.001")
+[step]
+--
+restoreDevonfwIde(["java","mvn"], "2020.08.001")
+--
+====
+Now, we will create a sample devon4j application with the name *sampleapp*.
+
+[step]
+--
+createDevon4jProject("sampleapp")
+--
+
+In command prompt just type "devon java migrate".
```

Comment:
Please contact @EduardKrieger on https://github.com/devonfw-tutorials/tutorial-compiler/pull/154 so you can use it here